adazza-dependency-management.gradle

```diff
@@ -37,6 +35,7 @@ allprojects {
         entry "finagle-thrift_$scalaVersion"
       }
 
+      // TODO: Resolve storehaus custom version lib dependencies
```

Comment:
This is actually not causing a problem. I initially confused this with  `scanamo-scrooge` and added this comment. This comment is now removed. 

I could still removed it if you'd like, but it should not be necessary since the lib dependencies (for both Scala 2.11 and 2.12) exists in S3.